dev-docs/bidders/connektai.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: bidder
+title: connekt.ai
+description: Prebid connekt.ai Bidder Adaptor
+biddercode: connektai
+media_types: banner, video
+coppa_supported: true
+tcfeu_supported: false
+usp_supported: true
+prebid_member: false
+pbjs: false
+pbs: true
+schain_supported: true
+floors_supported: true
+multiformat_supported: will-bid-on-any
```

Comment:
This document is missing a few pieces of adapter metadata. Please fill out the metadata as documented at https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter

The more, the better it is for publishers / ad tech vendors to vet your adapter